src/strands/agent/agent.py

```diff
@@ -135,13 +137,16 @@ def caller(
                     "name": normalized_name,
                     "input": kwargs.copy(),
                 }
+                tool_results: list[ToolResult] = []
+                invocation_state = kwargs
+
+                tool_executor = tool_executors.sequential.Executor(skip_tracing=True)
```

Comment:
I make a comment below, but it's a little weird to see `tool_executors.sequential.Executor` vs `tool_executors.SequentialExecutor`.  Long-term, for example, I could see us wanting to export this as `SequentialExecutor` from top-level strands module